packages/firebase_ml_vision/android/src/main/java/io/flutter/plugins/firebasemlvision/FirebaseMlVisionHandler.java

```diff
@@ -131,10 +132,16 @@ private FirebaseVisionImage dataToVisionImage(Map<String, Object> imageData) thr
       case "bytes":
         @SuppressWarnings("unchecked")
         Map<String, Object> metadataData = (Map<String, Object>) imageData.get("metadata");
-
+        ArrayList<Map<String, Object>> planeMetadata =
```

Comment:
Could you add tests for this?

Reply:
What type of test would you suggest? I looked into the flutter driver test, but mocking the necessary CameraImage seems hard.

Reply:
Gentle ping, any advice on this?